tests/testthat/test-guide-axis.R

```diff
@@ -382,6 +382,14 @@ test_that("guide_axis_theta sets relative angle", {
   expect_doppelganger("guide_axis_theta with angle adapting to theta", p)
 })
 
+test_that("guide_axis_theta with only one axis key", {
+  p <- ggplot(data.frame(x = "a", y = 1:3), aes(x, y)) +
+    geom_point() +
+    coord_radial(start = 0, end = pi / 2, inner.radius = 0.5) +
+    guides(theta = guide_axis_theta(angle = 0))
+  expect_doppelganger("guide_axis_theta with only one axis key", p)
+})
+
```

Comment:
I don't think this needs a visual snapshot, we can `get_guide_data(p, "theta")` to test the representation of the guide. The rest of the plot can also be simplified by leaving out the `guides()` and the arguments to `coord_radial()`.